src/main/resources/apotheosis.mixins.json

```diff
@@ -0,0 +1,15 @@
+{
+  "minVersion": "0.8",
+  "required": true,
+  "compatibilityLevel": "JAVA_8",
+  "refmap": "apotheosis.mixins.refmap.json",
+  "package": "shadows.apotheosis.mixin",
+  "mixins": [
+    "MixinBambooBlock",
+    "MixinCactusBlock",
+    "MixinSugarCaneBlock"
+  ],
+  "injectors": {
+    "defaultRequire": 1
```

Comment:
In addition to this, mixin injectors should include the max number of injections they are expected to make in a target (see argument `allow`). Alternatively, this option can be omitted entirely and both `require` and `allow` declared explicitly on the injectors.